### PR TITLE
trace-cmd: fix issue in post run hook

### PR DESCRIPTION
**Pull request by matsvanmolle**
This PR fixes the fact that the post_run_hook was an AsyncProcess. It is now possible to do post-dataprocessing on the output without race conditions. The process will now also not interfere with the experiment.

### CODE REVIEW

File: benchkit/commandattachments/tracecmd.py [hunk 1]
```diff
@@ -54,13 +54,13 @@ def post_run_hook(
         print(rdd)
 
         self._process.wait()
-
+
```
Comment (apaolillo) [inline benchkit/commandattachments/tracecmd.py:57]: revert that changes

File: benchkit/commandattachments/tracecmd.py [hunk 2]
```diff
@@ -54,13 +54,13 @@ def post_run_hook(
...
-            stdout_path=rdd / "generate-graph.out",
-            stderr_path=rdd / "generate-graph.err",
+                
+        output = self._platform.comm.shell(
+            command=command,
             current_dir=rdd,
+            print_output=False
         )
+        write_record_file_fun(output, "generate-graph.out")
+
```
Comment (apaolillo) [inline benchkit/commandattachments/tracecmd.py:66]: ensure only one linefeed at the end of the file

**Review by apaolillo — CHANGES_REQUESTED** [review]
thanks! Only small cosmetic changes required.

File: benchkit/commandattachments/tracecmd.py (hunk 1 above)
Comment (matsvanmolle) [reply benchkit/commandattachments/tracecmd.py]: Done

**Review by apaolillo — APPROVED** [review]

**Pull request merged**